store/src/file_store.rs

```diff
@@ -95,7 +95,7 @@ impl FileStore {
                                 None
                             }
                         })
-                        .filter(move |info| after.map_or(true, |v| info.timestamp >= v))
+                        .filter(move |info| after.map_or(true, |v| info.timestamp > v))
```

Comment:
iirc, doing this messed up the mobile rewards file fetch at the time, is there a particular reason the `>=` is being removed here?

Reply:
Yeah please do _not_ change semantics like that

Reply:
I have reverted this back to `>=` in 749fc0e. We can get back to it at later if we need to.